### PR TITLE
Edits to StPicoDstMaker.cxx and StPicoEvent.{h, cxx} to add new pico event level i…

### PR DESCRIPTION
**Pull request by BassamAboona**
…nformation

### CODE REVIEW

**Review by nigmatkulov — APPROVED** [review]
All good.

File: StRoot/StPicoEvent/StPicoEvent.h [hunk 1]
```diff
@@ -460,7 +508,31 @@ class StPicoEvent : public TObject {
...
+  Int_t  mEarliestVpdWHit;
+  /// Closest VPDE hit to start time
+  Int_t  mClosestVpdEHit;
+  /// Closest VPDW hit to start time
+  Int_t  mClosestVpdWHit;
+  /// Furthest VPDE hit compared to start time
+  Int_t  mLatestVpdEHit;
+  /// Furthest VPDW hit compared to start time
+  Int_t  mLatestVpdWHit;
+
```
Comment (plexoos) [inline StRoot/StPicoEvent/StPicoEvent.h:535]: Should ClassDef version increase when new members are added?

Comment (BassamAboona) [reply StRoot/StPicoEvent/StPicoEvent.h:535]: Yeah! I am pretty sure that it should be if the .h file is edited. I was not sure if it is something that I should do on my end, or if it is something the code owners do.

Comment (BassamAboona) [reply StRoot/StPicoEvent/StPicoEvent.h:535]: I would be more than happy to do it. Just let me know your preference.

Comment (nigmatkulov) [reply StRoot/StPicoEvent/StPicoEvent.h:535]: Hi Bassam, please do it.

Comment (BassamAboona) [reply StRoot/StPicoEvent/StPicoEvent.h:535]: No problem! Should I increment only the first ClassDef in the following from StPicoEvent.h?

Screen Shot 2021-06-15 at 9 12 52 AM

Comment (nigmatkulov) [reply StRoot/StPicoEvent/StPicoEvent.h:535]: Please do it for both.

File: StRoot/StPicoEvent/StPicoEvent.h [hunk 2]
```diff
@@ -154,7 +154,31 @@ class StPicoEvent : public TObject {
   Int_t    nTofT0() const                { return (Int_t)mNTofT0; }
   /// Return z position of the primary vertex estimated by VPD
   Float_t  vzVpd() const                 { return mVzVpd; }
-
+  // In the following, the factors of 1000. serve to convert times back and forth between external nsec and internal psec
+  /// Return number of particles before T0 outlier rejection
+  Int_t    nTofT0Can() const		 { return (Int_t)mNTofT0Can; }
+  /// Return event start time
```
Comment (veprbl) [inline StRoot/StPicoEvent/StPicoEvent.h]: Mark units please
```suggestion
  /// Return event start time in nano seconds
```
and below

File: StRoot/StPicoEvent/StPicoEvent.h [hunk 3]
```diff
@@ -154,7 +154,31 @@ class StPicoEvent : public TObject {
...
+  Double_t  tStart() const                { return ((Double_t)mTStart)/1000.; }
+  /// Return earliest candidate particle in T0 calculation
```
Comment (veprbl) [inline StRoot/StPicoEvent/StPicoEvent.h]:
```suggestion
  /// Return hit time in nano seconds for the earliest candidate particle in the T0 calculation
```

**Review by veprbl — COMMENTED** [review]
I have some comment nitpicks

**Review by plexoos — APPROVED** [review]

**Pull request merged**